bmorph/core/workflows.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 import xarray as xr
 from functools import partial
-from tqdm.autonotebook import tqdm
```

Comment:
Why this change? Does this break notebook usage?

Reply:
I think the issue with this is that importing the notebook version of tqdm then doesn't work for scripts. And I think vice versa is true, that then you need something like line 6 for the tqdm to work in notebooks. I'm not certain the best way to handle this

Reply:
The `tqdm.autonotebook` was supposed to be the cross-compatible version - I'll look into this.

Reply:
oop, I'm wrong nevermind